roles/kubernetes/master/tasks/kubeadm-setup.yml

```diff
@@ -103,7 +103,7 @@
 
 - name: kubeadm | Initialize first master
   command: >-
-    timeout -k 600s 600s
+    timeout -k 300s 300s
```

Comment:
Shouldn't timeout values be defined as variables as well?

Reply:
it's 5m internally hardcoded too. let's not worry about it for now since 5m is a good number